src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
```

Comment:
I think that's more dangerous though - it means that if you ever see that variable, you assume its the property, even though there's no guarantee, it's just assuming that people have followed a convention, which can't be strictly enforced anyway.

I know what you mean - but I think it's better to need to explicitly check the observers, or change the function name so it's purpose is clearly stated to observe a prop. In general, I'm a bit off the magic of complex observers.